File: app/src/main/java/com/bevstudio/wolfbooksapp/model/api/ReadingModes.java
package com.bevstudio.wolfbooksapp.model.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ReadingModes {

    @SerializedName("text")
    @Expose
    private Boolean text;
    @SerializedName("image")
    @Expose
    private Boolean image;

    public Boolean getText() {
        return text;
    }

    public void setText(Boolean text) {
        this.text = text;
    }

    public Boolean getImage() {
        return image;
    }

    public void setImage(Boolean image) {
        this.image = image;
    }

}

File: app/src/main/java/com/bevstudio/wolfbooksapp/model/api/ReadingModes.kt
package com.bevstudio.wolfbooksapp.model.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ReadingModes {
    @SerializedName("text")
    @Expose
    var text: Boolean? = null

    @SerializedName("image")
    @Expose
    var image: Boolean? = null
}